tslint.json

```diff
@@ -146,10 +146,6 @@
       "allow-fast-null-checks",
       "should"
     ],
-    "no-unused-variable": [
-      true,
-      "react"
-    ],
```

Comment:
Note: I also got the warning about this rule. In typescript 3.0.1 it is auto supported by the compiler.